ocaml/xapi/dbsync_slave.ml

```diff
@@ -89,6 +89,9 @@ let refresh_localhost_info ~__context info =
   debug "Updating host software_version and updates_requiring_reboot" ;
   Create_misc.create_updates_requiring_reboot_info ~__context ~host ;
   Create_misc.create_software_version ~__context ~info:(Some info) () ;
+  (* set startup policy in xenopsd *)
+  Xapi_host.set_numa_affinity_policy ~__context ~self:host
```

Comment:
The function `refresh_localhost_info` is meant to "Update the information in the Host structure". This new call is rather the opposite and is not really a good fit here.

Reply:
Also, `Xapi_host.set_numa_affinity_policy` does another DB-set inside of it, which is a bit of a waste. It would be nicer to split that function in a DB-set part and a call-xenopsd part.

Reply:
I moved this to Xapi_xenopsd and on_xapi_startup now.